src/fides/api/ops/schemas/saas/saas_config.py

```diff
@@ -298,21 +299,20 @@ def get_graph(self) -> Dataset:
         """Converts endpoints to a Dataset with collections and field references"""
         collections = []
         for endpoint in self.endpoints:
-            fields = []
-            for param in endpoint.requests["read"].param_values or []:
-                if param.references:
-                    references = []
-                    for reference in param.references:
-                        first, *rest = reference.field.split(".")
-                        references.append(
-                            (
-                                FieldAddress(reference.dataset, first, *rest),
-                                reference.direction,
-                            )
-                        )
-                    fields.append(ScalarField(name=param.name, references=references))
-                if param.identity:
-                    fields.append(ScalarField(name=param.name, identity=param.identity))
+            fields: List[Field] = []
+            read_request = endpoint.requests.get("read")
+            delete_request = endpoint.requests.get("delete")
+            if read_request:
+                self._process_param_values(fields, read_request.param_values)
+            elif delete_request:
+                # If the endpoint only specifies a delete request without a read,
+                # then we must use the delete request's param_values instead.
+                # One of the fields must automatically be flagged as a primary key
+                # in order for the deletion to execute. See fides#1199
+                self._process_param_values(fields, delete_request.param_values)
+                if fields:
+                    fields[0].primary_key = True
```

Comment:
For the example in this PR, could you alternatively define a `people` collection in the `saas_example_dataset.yml` with a single email field that is designated as the primary key?  It seems like there could be legitimate examples where there is already a dataset defined that will be merged with this config where a different field is a primary key.

I see that this primary key dependence will be dropped though for saas connectors which I think is a good call.

Reply:
I didn't add the `people` collection to the dataset since I didn't expect that to be defined for a delete-only collection but I can add it to make sure there's no issues with multiple fields marked as primary.